test/jdk/java/nio/channels/FileChannel/FileExtensionAndMap.java

```diff
@@ -117,6 +125,29 @@ public static void main(String args[]) throws Exception {
         }
     }
 
+    private static void checkRequiredDiskSpace(String destinationFolder) throws IOException {
+        Path destinationPath = Path.of(destinationFolder);
+        Path tmpDir = Path.of(TMPDIR);
+
+        long totalDiskSpaceNeeded = FILESIZE * PARALLELISM;
+
+        if (Files.getFileStore(tmpDir).name().equals(Files.getFileStore(destinationPath).name())) {
```

Comment:
I think you should be using FileStore::equals rather than equality of the name here.

Reply:
Done